### ISSUE TITLE
Prefix for classes

### ISSUE THREAD

**Issue by makasim** [fxb/php-last.fm-api#1]
It would be better if all classes in the library have prefix like lfCaller or like PEAR Services_LastFm_Caller. 
In this way we'll avoid a lot of troubles with class names conflicts.

What do you think?

**Comment by fxb** [comment]
Yeah, you're right. I also thought of that, but never changed it. I think I'll go with "LastfmCaller", "LastfmAlbum", etc.

**Comment by m4tthumphrey** [comment]
You could just use namespaces.

**Comment by fxb** [comment]
true. don't want to restrict it to PHP >= 5.3 though.

**Comment by m4tthumphrey** [comment]
Yeh, agreed. Maybe include an example of how to use with namespaces.

**Comment by patrickgalbraith** [comment]
I created a fork that is namespaced with LastFM_ ie LastFM_Artist etc... 
https://github.com/patrickgalbraith/php-last.fm-api

I also added the Chart api and changed the __autoload function to use spl_autoload_register() to avoid conflicts.

**Comment by Cruel** [comment]
I have a PSR-0 compliant namedspaced fork here for those interested:
https://github.com/Cruel/php-last.fm-api

It also has a composer.json for easy installation and autoloading in many workspaces.
